src/assets/stylesheets/components/experience.sass

```
@@ -0,0 +1,128 @@
+.experience
+  background-image: linear-gradient($color-white 0, $color-gray-lighter 100%)
+  padding: $m-3xl $m-base
+
+  .header
+    margin-bottom: $m-base
+
+    &:before
+      margin: $m-lg 0
```

Be specific, you only want to move to the left, so do only that
```suggestion
      margin-left: 0
```